wiki/forms.py

```diff
@@ -220,6 +222,10 @@ def clean_slug(self):
         slug = self.cleaned_data['slug']
         if slug.startswith("_"):
             raise forms.ValidationError(_(u'A slug may not begin with an underscore.'))
+
+        is_numeric = bool(re.match('^[0-9]+$', slug))
+        if is_numeric:
+            raise forms.ValidationError(_(u'Slug name cannot contain only digits'))
```

Comment:
Does the user specify the slug through the form? Isn't the slug an auto-generated field?

Reply:
It is auto-generated field, but could be edited